bsp_diff/caas/hardware/intel/external/drm-intel/0001-libdrm-changes-for-rpl-s.patch

```diff
@@ -26,19 +26,20 @@ index c00ac546..03ad7105 100644
 +	INTEL_VGA_DEVICE(0xA78A, info), \
 +	INTEL_VGA_DEVICE(0xA78B, info)
  
```

Comment:
There is an overlap in ID between RPL-S and RPL-P:

```
/* RPL-S */
#define INTEL_RPLS_IDS(info) \
        INTEL_VGA_DEVICE(0xA780, info), \
        INTEL_VGA_DEVICE(0xA781, info), \
        INTEL_VGA_DEVICE(0xA782, info), \
        INTEL_VGA_DEVICE(0xA783, info), \
        INTEL_VGA_DEVICE(0xA788, info), \
        INTEL_VGA_DEVICE(0xA789, info), \
        INTEL_VGA_DEVICE(0xA78A, info), \
        INTEL_VGA_DEVICE(0xA78B, info)

/*RPL-P*/
#define INTEL_RPLP_IDS(info) \
       INTEL_VGA_DEVICE(0xA7A0, info), \
       INTEL_VGA_DEVICE(0xA720, info), \
       INTEL_VGA_DEVICE(0xA7A8, info), \
       INTEL_VGA_DEVICE(0xA7A1, info), \
       INTEL_VGA_DEVICE(0xA721, info), \
       INTEL_VGA_DEVICE(0xA7A9, info), \
       INTEL_VGA_DEVICE(0xA780, info), \
       INTEL_VGA_DEVICE(0xA781, info), \
       INTEL_VGA_DEVICE(0xA782, info), \
       INTEL_VGA_DEVICE(0xA783, info), \
       INTEL_VGA_DEVICE(0xA788, info), \
       INTEL_VGA_DEVICE(0xA789, info), \
       INTEL_VGA_DEVICE(0xA78A, info), \
       INTEL_VGA_DEVICE(0xA78B, info)

```

Please check if this is expected.

Reply:
corrected and updated as per new Horizontal PR https://github.com/projectceladon/external-libdrm/pull/30/